### ISSUE TITLE
Window's default value should be dictionary object

### ISSUE THREAD

**Issue by taozuhong** [dmgbuild/dmgbuild#141]
The line 100 of `core.py` file will make an exception, the default value should return a dictionary structure
```python
settings["icon_size"] = json_data.get("icon-size", 80)
wnd = json_data.get("window", {"position": (100, 100), "size": (640, 480)})
pos = wnd.get("position", {"x": 100, "y": 100})
siz = wnd.get("size", {"width": 640, "height": 480})
settings["window_rect"] = (
    (pos.get("x", 100), pos.get("y", 100)),
    (siz.get("width", 640), siz.get("height", 480)),
)
```

**Comment by freakboy3742** [comment]
Thanks for the report; that's clearly a bug. Feel free to submit a pull request.

**Comment by HalfWhitt** [comment]
Looks like this was solved by #142 but never closed
